recipes/PICor/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "picor" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/PICor-{{ version }}.tar.gz
+  sha256: 2492f51600636aa4dbea90811e33aedce4824b16e0179676f8e492c0b6256cdd
+
+build:
+  entry_points:
+    - picor = picor.cli:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pyscaffold >=3.2a0,<3.3a0
+    - pip
+  run:
+    - python >=3.6
+    - pandas
+    - scipy
+    - docopt
+    - openpyxl
+
+test:
+  imports:
+    - picor
+  commands:
+    - pip check
+    - picor --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/MolecularBioinformatics/PICor
+  summary: Isotope correction for MS data
+  license: GPL-3.0
```

Proper SPDX identifier:

```suggestion
  license: GPL-3.0-only
```